board/safety.h

```diff
@@ -62,6 +62,12 @@ int safety_fwd_hook(int bus_num, CANPacket_t *to_fwd) {
   return (relay_malfunction ? -1 : current_hooks->fwd(bus_num, to_fwd));
 }
 
+bool get_longitudinal_allowed(void) {
+  int pedal_pressed = brake_pressed_prev && vehicle_moving;
+  pedal_pressed = pedal_pressed || gas_pressed_prev;
+  return controls_allowed && !pedal_pressed;
```

Comment:
Needs to be the same minus NO_DISENGAGE_ON_GAS check for the cars where we replace current_controls_allowed with this